packages/touchpoint-ui/src/App.tsx

```diff
@@ -39,7 +39,19 @@ export interface Props {
   config: Config;
   windowSize?: WindowSize;
   colorMode?: ColorMode;
+  /**
+   * Logo URL
+   * @deprecated - use brandIcon and launchIcon intead
```

Comment:
We haven't released and already there's something deprecated? 🙄

Reply:
You're right, I'll take it out!